integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -195,6 +195,7 @@ RUN case ${TARGETPLATFORM} in \
     --target linux-x64-rvc-ipv6only \
```

Comment:
Blocking comment, waiting for https://github.com/project-chip/connectedhomeip/pull/35195 to merge first